x-pack/filebeat/module/fortinet/firewall/ingest/pipeline.yml

```
@@ -82,14 +82,14 @@ processors:
     target_field: event.start
     formats:
     - UNIX_MS
-    timezone: "{{fortinet.firewall.tz}}"
+    timezone: "{{event.timezone}}"
     if: "ctx?.fortinet?.firewall?.eventtime != null && ctx.fortinet?.firewall?.tz != null && (ctx.fortinet?.firewall?.eventtime).length() > 11"
```

Should this check be changed now to `event.timezone` instead of `fortinet.firewall.tx`? (Same in the following `date` processors)
```suggestion
    if: "ctx?.fortinet?.firewall?.eventtime != null && ctx.event?.timezone != null && (ctx.fortinet?.firewall?.eventtime).length() > 11"
```